src/Game.py

```diff
@@ -19,10 +19,12 @@ def initGame(config1, config2, size):
         PLAYER_1: {
             "primary": getBoardFromConfig(player1Config),
```

Comment:
The key "primary" could be renamed to "fleet" or "board" (or another name)

Reply:
As the conception evolve, I think this will be delete in a future PR